tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -113,13 +114,22 @@ def layout():
                                         "Lower effect size",
                                         className='vp-text',
                                     ),
-                                    dcc.Input(
+                                    daq.Slider(
```

Comment:
You can use a `rangeSlider` for these two as well. Just a suggestion, not super important to implement :)

Reply:
Really? `module 'dash_daq' has no attribute 'rangeSlider'` so we'd have to define it...

Reply:
Whoops, made a typo. It's actually a `dcc` component: https://dash.plot.ly/dash-core-components/rangeslider

Reply:
Ok, thanks!

Reply:
Unfortunately, it's not as easy as replacing `daq.Slider` with `dcc.RangeSlider`... I'm not able to integrate this [syntax](https://dash.plot.ly/dash-core-components/rangeslider) into the current design because I can't troubleshoot: Web app won't render and there's no error stream...

Reply:
Is there anything showing in the console? Usually, if the web app isn't rendering, it's due to an invalid input ID. 

My suggestion is, instead of having two `daq.Slider` components, have a single `dcc.RangeSlider` and in the callback that uses lower/upper bounds (https://github.com/plotly/dash-bio/blob/5cb2ac5cce044cc8e394217417f7b834691b7d7b/tests/dashbio_demos/app_volcano_plot.py#L264), replace the individual `daq.Slider`s with the single `RangeSlider`.

Reply:
Well, there's the expected
```
TypeError: Unexpected keyword argument `handleLabel`
Allowed arguments: allowCross, className, count, disabled, dots, id, included, loading_state, marks, max, min, pushable, step, updatemode, value, vertical
```
but after dealing with that, no more error stream... blank, void...

Reply:
> instead of having two `daq.Slider` components

I'm not sure what you mean. There are *three* Slider components, corresponding to:
![sliders](https://user-images.githubusercontent.com/2227806/55495933-42315080-563e-11e9-9640-20cad0ede258.png)

Reply:
> Is there anything showing in the console?

No; to clarify, that's what I mean by "no error stream."

Reply:
I meant the two corresponding to lower/upper bounds (makes the most sense to replace with a `RangeSlider`, in my opinion). Also (sorry for not having clarified this), I'm wondering if there's anything showing up in the *browser* console. Sometimes, errors will only pop up there. 

Like I mentioned, it's not super critical to change the sliders -- if you're finding that it's taking too much time, you could leave it alone!

Reply:
Oh, I understand what you mean now!! I will leave it aside for this PR but keep it in mind for the future.